WordPress/Classes/Login/LoginWithUrlView.swift

```diff
@@ -89,26 +116,28 @@ struct LoginWithUrlView: View {
     }
 }
 
+extension LoginWithUrlView {
```

Comment:
Please make it `private`.

I think I like this approach more than `Strings`, but just noting that it's non-standard.